Add a warning for Keycloak URL changes after 8.5 (#4294)
camunda/camunda-docs: docs/self-managed/identity/user-guide/configuration/connect-to-an-existing-keycloak.md

## Steps

:::caution Keycloak URLs

> As of the 8.5 release, Identity uses the Keycloak frontend URL instead of the backend URL. This change may affect customers who have blocked the Keycloak frontend URL from other services, including Camunda applications, and can potentially impact Identity's functionality.

Reviewer: 💭 Just a consideration here, the change was made specifically in 8.5.3 so not sure if we should make this notice more specific?

> As of the 8.5 release, Identity uses the Keycloak frontend URL instead of the backend URL. This change may affect customers who have blocked the Keycloak frontend URL from other services, including Camunda applications, and can potentially impact Identity's functionality.

Reviewer 2: ```suggestion
As of the 8.5 release, Identity uses the Keycloak frontend URL instead of the backend URL. This change may affect you if you have blocked the Keycloak frontend URL from other services (including Camunda applications), and can potentially impact Identity's functionality.
```
Suggestion to address the customer directly in second person, rather than in the third person.
